reports-scheduler/src/test/kotlin/org/opensearch/integTest/PluginRestTestCase.kt

```diff
@@ -61,7 +61,7 @@ abstract class PluginRestTestCase : OpenSearchRestTestCase() {
     open fun wipeAllODFEIndices() {
         if (preserveODFEIndicesAfterTest()) return
         val response = client().performRequest(Request("GET", "/_cat/indices?format=json&expand_wildcards=all"))
-        val xContentType = XContentType.fromMediaTypeOrFormat(response.entity.contentType.value)
+        val xContentType = XContentType.fromMediaType(response.entity.contentType.value)
```

Comment:
this change came from opensearch core?

Reply:
yeah, due to upgrade in gradle version, this change is required for successful build